### PR TITLE
Components

### PR DESCRIPTION
**Pull request by dazehinn**
Features in this PR
* I create a new directory called components. 
* I create a calculator component inside the components directory
* I implement the design using only (JSX and CSS)
* I import the calculator component and use it in the top-level component: App.js

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,39 +1,60 @@
+body {
```
Comment (GzimAsani) [inline src/App.css]: - [ ] All the boilerplate that got created by the CRA (texts, images, and styles) should be deleted, lie the images and logos on public folder

File: src/components/Calculator.js [hunk 2]
```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```
Comment (GzimAsani) [inline src/components/Calculator.js:1]: - [ ] According to the requirements Calculator.js should have at least 1 child component

File: src/components/Calculator.js [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+import './Calculator.css';
```
Comment (GzimAsani) [inline src/components/Calculator.js:5]: - [ ] You do not have a Calculator.css and this is throwing an error, kindly fix it 
image

File: src/App.css [hunk 4]
```diff
@@ -1,39 +1,60 @@
...
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
+.Calculator {
+  display: grid;
+  justify-content: center;
+  align-content: center;
+  grid-template-columns: repeat(4, 60px);
+  grid-template-rows: repeat(6, 60px);
+  height: 100vh;
```
Comment (GzimAsani) [inline src/App.css]: - [ ] Kindly fix these borders in order to match the design better 
image

**Review by GzimAsani — CHANGES_REQUESTED** [review]
### Hello, @dazehinn               :wave:   
 Good job so far,  done There are some issues that you still need to work on to go to the next project but you are almost there!
 
 ## Highlights
 * Good job creating the project with create-app :+1: 

 ## Required Changes 

- [ ] In your README file kindly mention running the npm install
- [ ] Check the comments below :arrow_double_down: 

 ## Optional suggestions
 _Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding! clapclapclap
 
 Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.
 
 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ridwanediallo — CHANGES_REQUESTED** [review]
# Changes Required ♻️
## Hi @dazehinn, 


AlmostThereReachGIF

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- All linters checks are ok. ✔


 ## Required Changes ♻️

- [ ] Excellent work in implementing some of the changes suggested by the previous reviewer :clap: . However, I believe you have forgotten to remove the boilerplate. Please remove them; not only is it a requirement, but it also gives your project a more professional appearance and makes it easier to navigate. This :point_down:  is what we have so far.
Screenshot from 2023-02-24 22-40-31

- [ ] Please in the public folder remove these :point_right:  `logo192.png` `logo512.png` `manifest.json` `robots.txt`.
- [ ] In the src folder remove these :point_right: `App.test.js` `App.css` `index.css` `reportWebVitals.js` `logo.svg`.👌😊


## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dazehinn** [comment]
@ridwanediallo 
Thanks for pointing out the files to remove. I didn't know exactly which to remove since they all seemed important

**Review by ridwanediallo — APPROVED** [review]
# STATUS: APPROVED 🟢
## Hi @dazehinn, 
Congratulations!GIF

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- GitHub flow is used correctly. ✔
- Your commit message are good. ✔

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by dazehinn** [comment]
Oh, @ridwanediallo  it's you still. This is the first time I experience this.

**Pull request merged**